bundles/CoreBundle/Templating/LegacyTemplateGuesser.php

```diff
@@ -0,0 +1,146 @@
+<?php
+
+namespace Pimcore\Bundle\CoreBundle\Templating;
+
+use Psr\Container\ContainerInterface;
+use Sensio\Bundle\FrameworkExtraBundle\Templating\TemplateGuesser as BaseTemplateGuesser;
+use Symfony\Component\HttpKernel\KernelInterface;
+use Doctrine\Common\Persistence\Proxy;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Bundle\FrameworkBundle\Templating\TemplateReference;
+
+/**
+ * Provides backward compatibility for camelCase template names and PHP engine support
+ *
+ */
+class LegacyTemplateGuesser extends BaseTemplateGuesser
+{
+    /**
+     * @var KernelInterface
+     */
+    private $kernel;
+
+    /**
+     * @var $container
+     */
+    private $container;
+
+    /**
+     * @var string[]
+     */
+    private $controllerPatterns;
+
+    public function __construct(KernelInterface $kernel, ContainerInterface $container, $controllerPatterns = [])

```

Comment:
is it really necessary to pass in the container here? when you only need 'templating' from it?

Reply:
@dvesh3 can you please fix that? Thanks :)

Reply:
@brusch done :)

Reply:
cool :), I guess the TemplateGuesser and and TemplateListener is for the auto rendering right?

Reply:
yup, Guesser generates the tempate path and Listener is responsible for rendering it.